manager_local.go

```diff
@@ -15,14 +15,6 @@ func NewLocalManager() Manager {
 	}
 }
 
-func NewLocalManagerBlockingProcesses() Manager {
```

Comment:
why delete?

Reply:
This was only here because we used to have `basicProcess`, and so there was a need to distinguish a normal `NewLocalManager` and a `NewLocalManagerBlockingProcesses`. Since we removed `basicProcess`, this distinction is no longer necessary.